test/_init_tests.js

```diff
@@ -188,6 +188,10 @@ afterEach(() => {
     sandboxes.splice(0, sandboxes.length).forEach(sb => sb.restore());
     throw new Error('You forgot to restore your sandbox!');
   }
+  if (global.sandbox) {
+    delete global.sandbox;
```

Comment:
Cleanup is not really a goal here: it's an exception and should be fixed. In other words, there should be exactly zero cases where `global.sandbox` is not cleared.